packages/sdks/javascript/README.md

```diff
@@ -89,22 +89,27 @@ To build and test the SDK locally, [**fork this repository**](https://github.com
   xcode-select --install
   ```
 ### Setup and run in local environment
-#### 1. Build Docker Image
+#### 1. Navigate to the project directory!
```

Comment:
I believe the exclamation mark here brings to the reader the idea of a very strong feeling being expressed.
As here we have a set of instruction with simple and clear steps, that might not be the most appropriate "voice tone" to be used here.

I'm offering a suggestion below to be quickly applied if you agree with it:
```suggestion
#### 1. Navigate to the project directory
```

Reply:
Indeed, that is a very good point! I believe the same step on "Without docker" on the tutorial should also receive the same modifications.


```suggestion
#### 1. Navigate to the project directory
```

Reply:
Indeed, that is a very good point! I believe the similar step on "Without docker" on the tutorial should also receive the same modifications.

Reply:
@Vitor-the-Dev feel free to also modify the other step you mentioned if you feel it fits in this same PR.